src/core/energy_storage/electric_storage.jl

```diff
@@ -274,6 +273,10 @@ struct ElectricStorage <: AbstractElectricStorage
     function ElectricStorage(d::Dict, f::Financial)  
         s = ElectricStorageDefaults(;d...)
 
+        if s.dc_coupled && s.off_grid_flag
```

Comment:
Curious what the main limitation is here? It seems to me like off-grid is the biggest use case for this!

Reply:
only that I ran out of time to modify the off grid function add_operating_reserve_constraints :(